src/content/pages/explore.mdx

```diff
@@ -6,136 +6,129 @@ subtitle: At EuroPython, many attendees will come with their families. While not
+- [Eateria](https://maps.app.goo.gl/52VnUHg1C8qWkSjC8): Known for vegan junk food - best burger
+- [Pastva](https://maps.app.goo.gl/7tux1PYfSKcAdv6F7): Open during the day, serving European and healthy food
+- [Dhaba Beas](https://dhababeas.cz/en/): Offers cheap buffets, with Indian and European dishes, widely available across Prague.
+- [Share Sweet & Espresso Bar](https://www.sharesweetbar.cz/en): Coffee shop with the best vegan cakes
 
-According to HappyCow, there are 89 vegan and vegetarian places to eat. So it should not be an issue to find something. Typical Czech restaurants mostly only offer fried cheese and French fries, sometimes pickled camembert. Czech cuisine is mostly meat-heavy. Hungry during the night? Go to any kebab place and you can order a falafel wrap with veggies. You can also go to Burrito Loco - it’s a Mexican food chain and they offer vegan mana burrito and burrito with grilled veggies.
+According to HappyCow, there are 89 vegan and vegetarian places in Prague. So it should not be an issue to find something. Czech restaurants typically only offer fried cheese and French fries, occasionally including pickled camembert as vegetarian/vegan options, given Czech cuisine's heavy emphasis on meat.
 
-[Visit HappyCow vegan-vegetarian listings](https://www.happycow.net/europe/czech_republic/prague/?filters=vegan-vegetarian)
+Hungry during the night? Go to any kebab place and order a falafel wrap with veggies. You can also go to Burrito Loco - a Mexican food chain and they offer vegan mana burrito and burrito with grilled veggies.
+
+[Visit HappyCow's vegan-vegetarian listings](https://www.happycow.net/europe/czech_republic/prague/?filters=vegan-vegetarian)
 
 ## Where to drink?
 
-In European countries, drinking is a popular way to socialize. Here are some recommendations:
+In European countries, it is popular to socialise over drinks (though there's no pressure to consume alcohol). Here are some recommendations:
 
-- For craft beer: Sibeeria Bar, Pivotéka, BeerGeek, Zlý časy
+- For craft beer: [Sibeeria Tap Room](https://maps.app.goo.gl/PrRja7hbprfnYqKN8), Pivotéka, BeerGeek, [Zlý časy](https://maps.app.goo.gl/rCYojfxCJuXwZicAA)
 - Pilsner-type beer: everywhere
-- For mild and not bitter Pilsner-style beer, try Únětice (it’s in a few places). They also have 8 degrees with less alcohol
-- For more bitter, try Plzeň/Pilsner or Kozel, Radegast
-For craft, try Matuška (APA, IPA, ALE), Zíchovec (Sour ALE, APA, IPA, ALE) or Sibeeria (Sour ALE, APA, IPA, ALE)
-- For wine: Bublinka 2024 wine festival
-
-On top of the above:
+  - For mild and non-bitter Pilsner-style beer, try Únětice (available in a few places). They also offer an 8-degree one with lower alcohol content.
+  - For more bitter options, try Plzeň/Pilsner, Kozel or Radegast.
+  - For craft beers, try Matuška (APA, IPA, ALE), Zíchovec (Sour ALE, APA, IPA, ALE) or Sibeeria (Sour ALE, APA, IPA, ALE)
```

Comment:
For people who know beers, is the craft beers section here part of the Pilsner? it feels very strange to me. If not, it should be merged to the first bullet point of craft beer, no?

Reply:
there are different types of beer:
* lager
  * its subsection is pilsner style, which was originally created in Pilsner city and the brewery and beer is called Pilsner as well (confusing, I know)
  note: Pilsner beer (not style, but that specific one) is very bitter - 40 IBU and if you want a pilsner style beer, but less bitter, then I recommend únětice beer
* craft beers:
  * ALE 
    * sour ale
    * APA (American pale ale)
    * IPA (Indian pale ale)

Reply:
these are the most popular ones. there are beers like stout etc., but that's not brewed here. they mostly offer different kinds of beer in specialized beer places and it's always imported bottled beer. so if you're into beer, I recommend trying something local and draught beer - avoid bottled ones

Reply:
Hmm it's just that in the text, here it already listed craft beer: https://github.com/EuroPython/website/blob/b3b96db3fc33d4b3b178d8b1d10db04c81e29fcc/src/content/pages/explore.mdx?plain=1#L107 So should I just do something like this? 

- For craft beer: Try [Sibeeria Tap Room](https://maps.app.goo.gl/PrRja7hbprfnYqKN8) (Sour ALE, APA, IPA, ALE), Pivotéka, BeerGeek, [Zlý časy](https://maps.app.goo.gl/rCYojfxCJuXwZicAA), Matuška (APA, IPA, ALE), or Zíchovec (Sour ALE, APA, IPA, ALE) 
- Pilsner-type beer: everywhere
  - For mild and non-bitter Pilsner-style beer, try Únětice (available in a few places). They also offer an 8-degree one with lower alcohol content.
  - For more bitter options, try Plzeň/Pilsner, Kozel or Radegast.

Reply:
LGTM!

Reply:
This is applied in https://github.com/EuroPython/website/pull/748/commits/d5170535912668ad3f56ff2015c1ed834a94e3c7. Thank you so much for your help to someone who cannot drink beer! :hugs: